widgetastic_manageiq.py

```diff
@@ -875,6 +876,27 @@ def first_page(self):
     def last_page(self):
         self.paginator.last_page()
 
+    def pages(self):
```

Comment:
awesome. this thing came to my mind then but I prefer to new code only when it is really necessary.